workflow/executor/executor.go

```diff
@@ -794,19 +793,38 @@ func (we *WorkflowExecutor) waitMainContainerStart() (string, error) {
 	}
 }
 
+func watchFileChanges(ctx context.Context, pullTimeout time.Duration, filePath string) <-chan struct{} {
```

Comment:
Can we name the variable: `pollInterval`

Reply:
Sure. thank you for suggesting better name